x-pack/plugins/task_manager/server/routes/background_task_utilization.ts

```diff
@@ -111,6 +111,12 @@ export function backgroundTaskUtilizationRoute(
     router.get(
       {
         path: `/${routeOption.basePath}/task_manager/_background_task_utilization`,
+        security: {
+          authz: {
+            enabled: false,
+            reason: 'This route is opted out from authorization',
```

Comment:
@cnasikas @adcoelho could we please elaborate on reasoning for this one?

Reply:
cc @elastic/response-ops-execution

Reply:
From: https://github.com/elastic/kibana/issues/155778#issuecomment-1631365818

It authorizes via JWT and is the required way to have the metrics collected via the k8s daemonset. So we are ok with this one 👍